### PR TITLE
Add docs for nodeunit Bamboo transformer

### PR DESCRIPTION
**Pull request by Chaseshak**
## What's changing?
* Add docs for transformer in github/valet#6328

## How's this tested?
* Docs only

Closes github/valet#6296

### CODE REVIEW

**Review by j-dunham — APPROVED** [review]

**Review by simonsanchez — APPROVED** [review]

File: docs/bamboo/index.md [hunk 1]
```diff
@@ -33,7 +34,7 @@
 | [Stop Job](plugins/StopJob.md)                   | run                                                               |
 | [TestNG](plugins/TestNG.md)                      | scacap/action-surefire-report                                     |
 | [Unlock Keychain](plugins/UnlockKeychain.md)     | run                                                               |
-| [Visual Studio](plugins/VisualStudio.md)        | seanmiddleditch/gha-setup-vsdevenv, run                           |
+| [Visual Studio](plugins/VisualStudio.md)         | seanmiddleditch/gha-setup-vsdevenv, run                           |
```
Comment (Mjefoester) [inline docs/bamboo/index.md:37]: thanks

Comment (Mjefoester) [reply docs/bamboo/index.md:37]: thanks

Comment (Mjefoester) [reply docs/bamboo/index.md:37]: > thanks

**Review by Mjefoester — COMMENTED** [review]
😀

**Comment by jjfiur** [comment]
n2258uwrt[9]tgv brv wwopttertg ioweroopvtob[rgtgepty9erv ut8ytu ber;orteet[t[3]t3[0tt]te08344t0909ergoiv3t

**Comment by jjfiur** [comment]
i 3uittoet]p34ty]etioter0945wuerg45895ty4[o[''
tp
4
y455y y]455y

**Comment by jjfiur** [comment]
08b75yoitgpt34tewt u 54ipigejkk ipw
ertt]t\\5t]t\t\4y4y9n4eruu4595et vdfhgdjkdgt]
-]9erge rvtppawe\--seey

**Pull request merged**